backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java

```diff
@@ -14,11 +14,32 @@
 
 package com.google.sps.perspective;
 
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeType;
+
 /**
  * Handles analysis of stories for content and provides a recommendation on
  * the "best" appropriate story (if multiple).
  */
-interface PerspectiveManager {
+public interface PerspectiveManager {
```

Comment:
Nit: Interfaces are default public so no need to declare it.
(Thanks Joe ;) )

Reply:
Will change. Thanks!

Reply:
Done